src/contributors.js

```diff
@@ -41,6 +41,11 @@ export const sreeram = {
   username: "sreeram-venkitesh"
 };
 
+export const dx-dy-dz = {
```

Comment:
Could you change it to something else and commit again. This is not a valid identifieer. Also, this need not be your GitHub id. 
For instance, you can change it as:

```js
export const dx_dy_dz = {
```